pynuodb/session.py

```diff
@@ -197,14 +198,20 @@ def _parse_addr(self, addr, options):
                     if options['ipVersion'] == 'v6':
                         ver = 6
 
-                # get ip addr from hostname
-                af = socket.AF_INET
-                if ver == 6:
-                    af = socket.AF_INET6
-                try:
-                    ip = socket.getaddrinfo(host, port, af)[0][4][0]
-                except:
-                    raise SessionException("Addr lookup failed %s %d %d" % (host,port,af))
+                # get ip addr from hostname unless not running on Linux
```

Comment:
I dont see any unit tests for the function. How did you verify that it works correctly?

Reply:
I have been testing with test code in build/test/ipv6-docker which is presently only on my branch. Also there are some unit tests under Platform/tests.